lib/constants/general.dart

```diff
@@ -13,3 +14,33 @@ Measurement getLowest(List<Measurement> list) => list
 /// Get highest measurement in list
 Measurement getHighest(List<Measurement> list) => list
     .reduce((value, element) => value.value > element.value ? value : element);
+
+/// Define a "wrapper class" to be used by provider to find the bottomNavigationBar
+/// index up in the widget tree not confusing it with other simple integers that
+/// may be present
+class BottomNavigationBarIndex extends ChangeNotifier {
+  int _index;
+
+  BottomNavigationBarIndex(this._index);
+
+  int get index => _index;
+
+  set index(int index) {
+    _index = index;
+
+    notifyListeners();
+  }
+}
+
+/// Navigator paths of the homepage of paziente
+const List<String> kHomeNavigatorPaths = [
+  "home",
+  "agenda",
+  "notifiche",
+  "dottore",
+  "profilo"
+];
```

Comment:
Have you considered the pairing and feedback screens too? How they will be reached by the user?

Reply:
Pairing is on top right of the app bar (cgm icon), while for the feedback screen I completely forgout about it, maybe on the dottore screen? Since it's related to the doctor.

Reply:
On the dottore or agenda screen, maybe the first one is more appropriate imho.

Reply:
Yeah, pretty much the same since it's related to both things. Let's go for both lol